src/styles/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
```

Comment:
I think add "width: 200px;" great idea than adding "max-width". Because when width viewport will be smallest than the 200px we will have horizontal scroll, and we can read information.